quarkus-debezium-parent/quarkus-debezium-postgres-parent/deployment/src/main/java/io/quarkus/debezium/postgres/deployment/PostgresEngineProcessor.java

```diff
@@ -25,14 +35,35 @@
 
 public class PostgresEngineProcessor {
 
+    public static final String POSTGRESQL = "postgresql";
```

Comment:
Is this something that should be taken from `io.debezium.connector.postgresql.Module.name()`?